test/core.jl

```diff
@@ -1180,6 +1180,10 @@ let A = [1]
     @test x == 1
 end
 
+# Make sure that `Module` is not resolved to `Core.Module` during sysimg generation
```

Comment:
Is the comment accurate? Or it should be `Main.Module`?

Reply:
It is accurate